pages/events.jsx

```diff
@@ -8,12 +8,11 @@ var IconLinks = require('../components/icon-links.jsx');
 var IconLink = require('../components/icon-link.jsx');
 var Illustration = require('../components/illustration.jsx');
 var PageEndCTA = require('../components/page-end-cta.jsx');
+var util = require('../lib/util');
```

Comment:
Wondering if we should give this var a different name?  We've been assigning `require('util')` to `util`.
```javascript
var util = require('util');
```
found 3 occurrences